hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1092,18 +1090,23 @@ int SaraNcpClient::selectSimCard(ModemState& state) {
             } else if (r == 1 && static_cast<UbloxSaraUmnoprof>(umnoprof) == UbloxSaraUmnoprof::STANDARD_EUROPE) {
                 auto respBand = parser_.sendCommand(UBLOX_UBANDMASK_TIMEOUT, "AT+UBANDMASK?");
                 uint64_t ubandCatm1 = 0;
-                auto retBand = CHECK_PARSER(respBand.scanf("+UBANDMASK: 0,%llu", &ubandCatm1));
+                char ubandCatm1_str[24] = {};
+                auto retBand = CHECK_PARSER(respBand.scanf("+UBANDMASK: 0,%23[^,\n]", ubandCatm1_str));
```

Comment:
Nit: `\n` shouldn't be needed, Gen 3 parser will anyway strip it.

Reply:
I'm reading the R4 AT reference and it seems that `AT+UBANDMASK?` may give multiple strings. Should we guard for that?

![image](https://user-images.githubusercontent.com/388478/92642948-2d64ec00-f30b-11ea-8323-aca1f19e8663.png)

Reply:
Based on https://github.com/particle-iot/device-os/pull/2185/files#r485852743 this will need to get modified to read all the intermediate responses similar to https://github.com/particle-iot/device-os/blob/develop/hal/network/ncp_client/sara/sara_ncp_client.cpp#L702